scripts/ci/build-installers.sh

```diff
@@ -48,8 +48,8 @@ if [ "$ARGV_OPERATING_SYSTEM" == "linux" ]; then
   ./scripts/build/docker/run-command.sh \
     -r "$TARGET_ARCH" \
     -s "$(pwd)" \
-    -c 'make installers-all'
+    -c 'make electron-develop installers-all'
```

Comment:
Yes, the problem is that the sanity check that ensures the shrinkwrap file doesn't contain optional dependencies actually removes the optional dependencies from `node_modules`, which means that when we run the installers, some optional dependencies are not needed anymore.

We need find a good way to fix that, but as a quick fix, we can re-run `electron-develop`, which will only install the dependencies that are missing, before creating the installers.